packages/expo-notifications/src/Notifications.types.ts

```diff
@@ -183,11 +183,13 @@ export enum AndroidNotificationPriority {
   MAX = 'max',
 }
 
-export type NotificationContent = {
+export type NotificationContent<T = { [key: string]: unknown }> = {
```

Comment:
```suggestion
export type NotificationContent<T extends Record<string, unknown> = Record<string, unknown>>
```
I think we should constrain T to be only the `Record` type.

Reply:
I'll submit this changes today!